portfolio/src/main/webapp/index.html

```diff
@@ -37,6 +37,11 @@ <h3 id="Experience">Experience</h3>
      <h2 id="Contact">Contact Me</h2> 
      <p style="color:purple;">Bpaynes4291@gmail.com</p> 
      <a href="https://github.com/BlessP/Step"><p style="color:purple;">Github</p></a>
+     <div class="buttons" id="message">
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Click for my name!</button>
+      <div id="name-container"></div>
+      </div>
+     </div>
     </div>
```

Comment:
Actually, my bad, you didn't need an extra div. You can remove it, sorry about the churn!